code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -190,29 +190,60 @@
 	req_one_access = list(ACCESS_JANITOR, ACCESS_ROBOTICS)
 
 
-/mob/living/simple_animal/bot/cleanbot/get_controls(mob/user)
-	var/dat
-	dat += hack(user)
-	dat += showpai(user)
-	dat += text({"
-<TT><B>Cleaner v1.1 controls</B></TT><BR><BR>
-Status: []<BR>
-Behaviour controls are [locked ? "locked" : "unlocked"]<BR>
-Maintenance panel panel is [open ? "opened" : "closed"]"},
-text("<A href='?src=[UID()];power=1'>[on ? "On" : "Off"]</A>"))
-	if(!locked || issilicon(user) || user.can_admin_interact())
-		dat += text({"<BR>Cleans Blood: []<BR>"}, text("<A href='?src=[UID()];operation=blood'>[blood ? "Yes" : "No"]</A>"))
-		dat += text({"<BR>Patrol station: []<BR>"}, text("<A href='?src=[UID()];operation=patrol'>[auto_patrol ? "Yes" : "No"]</A>"))
-	return dat
-
-/mob/living/simple_animal/bot/cleanbot/Topic(href, href_list)
```

Comment:
Please test this with PDA bot controls. I believe those use radio signal `topic`s in the same way that the AI door control did, so be sure.

Reply:
There aren't any PDA controls for cleanbots. The nearest thing is the Custodial Locator PDA app the Janitor gets, but even there, all it does is show the bot's co-ords... it doesn't allow remote control of the bots.